cuebot/src/main/java/com/imageworks/spcue/servant/ManageFrame.java

```diff
@@ -280,10 +280,12 @@ public void addRenderPartition(FrameAddRenderPartitionRequest request,
                     .build());
             responseObserver.onCompleted();
         }
-        responseObserver.onError(Status.INTERNAL
-                .withDescription("Failed to find suitable frames.")
-                .augmentDescription("customException()")
-                .asRuntimeException());
+        else {
```

Comment:
nit: Let's move the `else` line up to the same line as the preceding closing bracket, so it looks like:

```
if {
  ...
} else {
  ...
}
```

Reply:
Done.